README.md

```diff
@@ -1,19 +1,289 @@
 # Cypress CircleCI Orb
 
-The Cypress CircleCI Orb is a piece of configuration set in your `circle.yml` file to correctly install, cache and run [Cypress.io](https://cypress.io) tests on CircleCI with very little effort. See this orb in [CircleCI Registry](https://circleci.com/orbs/registry/orb/cypress-io/cypress)
+The [Cypress CircleCI Orb](https://github.com/cypress-io/circleci-orb) is a
+piece of configuration set in your `.circleci/config.yml` file to correctly
+install, cache and run Cypress with very little effort.
 
-## Example
-```yml
-# to use orbs, must use version >= 2.1
+Full documentation can be found at the
+[`cypress-io/circleci-orb`](https://github.com/cypress-io/circleci-orb) repo.
+For the Orb Quick Start Guide and usage cases, view the CircleCI
+[Cypress orb documentation](https://circleci.com/developer/orbs/orb/cypress-io/cypress).
+
+---
+
+## Jobs
+
+### run
+
+A single, complete job to run Cypress tests in your application
+
+## Basic Setup
```

Comment:
We should add something indicating this is the out-of-the-box solution mentioned above.